pq2json/src/converter.rs

```diff
@@ -43,15 +43,15 @@ pub fn convert(
         None => Box::new(BufWriter::with_capacity(WRITER_BUF_CAP, io::stdout())) as Box<dyn Write>,
     };
 
-    let schema = settings
+    let (schema, missing_columns) = settings
         .columns
         .as_ref()
-        .map(|c| projected_schema(&reader, &c).unwrap());
+        .map(|c| projected_schema(&reader, &c).unwrap()).expect("Projected schema is not available.");
```

Comment:
I'm not sure we want to always require `--columns` parameter.

Reply:
Great catch, redone.